app/lib/settings/src/about_page/about_page_contributors.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:schulplaner8/Helper/GitHub.dart';
+import 'package:schulplaner8/Helper/helper_data.dart';
+import 'package:schulplaner8/Helper/helper_views.dart';
+import 'package:schulplaner_addons/schulplaner_utils.dart';
+import 'package:schulplaner_translations/schulplaner_translations.dart';
+import 'package:schulplaner_widgets/schulplaner_forms.dart';
+import 'package:schulplaner_widgets/schulplaner_theme.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+int contributorCount = 1;
+
+class AboutContributors extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: MyAppHeader(
+        title: bothlang(context, de: 'Mitwirkende', en: 'Contributors'),
+      ),
+      body: SingleChildScrollView(
+        child: Column(
+          children: [_AboutContributors(), _Contributors()],
+        ),
+      ),
+    );
+  }
+}
+
+class _AboutContributors extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return FormSection(
+        title: BothLangString(de: 'Mitwirkende am Schulplaner', en: 'Schoolplanner\'s collaborators').getText(context),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            FormSectionText(
+                text: DefaultTextSpan(
+              context,
+              BothLangString(
+                      de: 'Der Schulplaner ist Open-Source, sodass jeder daran mitwirken kann. Unten findet ihr eine Liste, welche dynamisch direkt von GitHub geladen wurde und alle Mitwirkenden beinhaltet! Klicke sie an, um zu ihrem GitHub Account zu gelangen!',
+                      en: 'The school planner is open source, so anyone can contribute to it. Below you can find a list, which was dynamically loaded directly from GitHub and includes all contributors! Click them to go to their GitHub account!')
+                  .getText(context),
+            )),
+            SizedBox(
+              height: 20.0,
+            )
+          ],
+        ));
+  }
+}
+
+class _Contributors extends StatelessWidget {
```

Comment:
This seems very hacky, what is the purpose of this class. I don't think that it is required to define contributorCount and define a height.

Reply:
The issue is that your are using two lists inside of each other. First the SingleChildScrollView and second the Listview. If you replace Listview with Column this issue should be fixed, as you won't have two scrollable Widgets.

Reply:
I don't think it is possible to replace the `ListView.builder` here

Reply:
Why?